### ISSUE TITLE
Content edits for #70274

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#71024]
## Info
Content edits according to [#70274](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70274)
## Tasks
- [x] Review support ticket 
- [x] Add, 'Local development (http://localhost:3000/flipper/features) (not authenticated)' as outlined in the ticket.
- [x] Explore adding list of endpoints to the feature toggles guide page. 
- [x] Confirm edits
- [x] Prepare page for deployment
- [x] Deploy page

## Acceptance Criteria
- [x] Page has been deployed
- [x] A comment has been provided in the original GitHub ticket with a final update on the status of the changes.

**Comment by taylojill** [comment]
I updated both pages as requested and sent two pdf docs for the requestor to review. 
Here's a summary of what I did:
1. On the Flipper UI Access page: I added "Local development" to Vets-API Flipper UI endpoints bulleted list.
2. On the Feature toggles guide page: I added the updated "Vets-API Flipper UI endpoints" list near top of page, under the Feature toggles list. Let me know if the placement of the list works or if it should be moved.

Waiting for feedback/approval from requestor.
